src/Raven.Studio/typescript/common/generalUtils.ts

```diff
@@ -32,6 +32,18 @@ class genUtils {
     static readonly urlRegex = /^(https?:\/\/)([^\s]+)$/; // allow any char, exclude white space
     static readonly invalidUrlMessage = "Url format expected: 'http(s)://hostName'";
     
+    static isValidUri(uri: string) {
```

Comment:
rename to Url, 
see: https://danielmiessler.com/study/difference-between-uri-url/

Reply:
done